src/uv-common.h

```diff
@@ -415,6 +415,7 @@ struct uv__iou {
   size_t sqelen;
   int ringfd;
   uint32_t in_flight;
+  int flags;
```

Comment:
```suggestion
  uint32_t flags;
```
Because `UV__MKDIRAT_SYMLINKAT_LINKAT` is unsigned.